pkg/ebpf/bytecode/tracer-ebpf.go

```diff
@@ -106,7 +106,7 @@ func tracerEbpfDebugO() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "tracer-ebpf-debug.o", size: 59632, mode: os.FileMode(436), modTime: time.Unix(1, 0)}
+	info := bindataFileInfo{name: "tracer-ebpf-debug.o", size: 59632, mode: os.FileMode(420), modTime: time.Unix(1, 0)}
```

Comment:
why did this change?

Reply:
It got updated when I ran `inv -e system-probe.build`.

Reply:
The mode was `420` before and got changed in a recent PR: https://github.com/DataDog/datadog-agent/commit/d80e0e01a5e0215f1a903139427e4ad9009626de#diff-27fefe35598678e96baca1073ca410f6R109. @p-lambert , any idea why it changed?

Reply:
FWIW, I am using latest `master` in `agent-dev_box`.

Reply:
If this is cool with you, it's cool with me :)